src/main/java/org/mutabilitydetector/checkers/CollectionWithMutableElementTypeToFieldChecker.java

```diff
@@ -54,7 +54,7 @@
     
     private final Map<String, String> fieldSignatures = newHashMap();
     private final AnalysisInProgress analysisInProgress;
-
+    
```

Comment:
Here and below: these unrelated whitespace changes can be reverted to minimize the diff.

Reply:
There is constructor definition after line 56, that is why the white space added or i might have removed it and then added it again when i was doing a code. i will do something to avoid the whitespace changes.